### ISSUE TITLE
We need a tooltip for the `Send` button/icon in the chat-panel textarea

### ISSUE THREAD

**Issue by stephendonner** [brave/brave-browser#33731]
## Description 

We need a tooltip for the `Send` button/icon in the chat-panel textarea

## Steps to Reproduce

1. install `1.61.32`
2. launch Brave
3. restart (to pick up Griffin seed)
4. click on `Brave Leo` in the sidebar
5. click on `Accept and begin`
6. type some text into the textarea below
7. hover over the paper airplane icon

## Actual result:

No tooltip

<img width="392" alt="Screen Shot 2023-10-18 at 1 57 11 PM" src="https://github.com/brave/brave-browser/assets/387249/a2934fbd-fd2d-4b8b-81d8-2302cda540cb">

## Expected result:

Perhaps `Send`?  Although I'll note this icon doesn't have the disabled state, so it won't always be accurate, technically speaking - separate nit, though

## Reproduces how often: 

100%

## Brave version (brave://version info)

```
Brave | 1.61.32 Chromium: 118.0.5993.88 (Official Build) nightly (x86_64)
-- | --
Revision | 8b2743769e6138ec51e617fd7641a9b0171a9172
OS | macOS Version 11.7.10 (Build 20G1427)
```

## Version/Channel Information:

- Can you reproduce this issue with the current release? `N/A`
- Can you reproduce this issue with the beta channel? `Yes`
- Can you reproduce this issue with the nightly channel? `Yes`

cc @bbondy @mattmcalister @petemill @nullhook @brave/qa-team

**Comment by stephendonner** [comment]
This is important for accessibility and usability.

**Comment by kjozwiak** [comment]
The above requires `1.60.105` or higher for `1.60.x` verification 👍

**Comment by GeetaSarvadnya** [comment]
Verification **PASSED** on

```
Brave | 1.60.105 Chromium: 119.0.6045.59 (Official Build) (64-bit)
-- | --
Revision | a39727aa81373a09a915478fcb35826242c37335
OS | Windows 10 Version 22H2 (Build 19045.3570)

```
Using the STR/Cases outlined via https://github.com/brave/brave-browser/issues/33731#issue-1950659877, ensured that the `Send` icon within the `Leo` chat has a tooltip once you hover over the icon as per the following:

**`Example`** | **`Example`**
----------------|-----------------
![image](https://github.com/brave/brave-browser/assets/38657976/35bf55ee-7fe4-4032-a17c-f25819d4fdd4) | ![image](https://github.com/brave/brave-browser/assets/38657976/813979f4-369a-4bf4-846d-331cd71beab6)
